tests/functional/list/test_commands.py

```diff
@@ -29,10 +29,11 @@
     "show",
     "snapshot",
     "freshness",
+    "serve",  # this was actually serving up docs and caused issues locally
 }
 
 # Commands to happy path test
-commands = [command.value for command in Command if command.value not in commands_to_skip]
+commands = [command.to_list() for command in Command if command.value not in commands_to_skip]
```

Comment:
I was confused how `command.to_list()` worked in converting an enum entry into a list like `['build']`, but it does! Cool! Weird, but cool.